tests/test.hs

```diff
@@ -534,12 +540,17 @@ mkTest ec dir file
     test = dir </> file
 
 
+logFilePath :: FilePath -> FilePath -> FilePath
+logFilePath dir file = "tests/logs/cur" </> testFileFullPath <.> "log"
+  where
+    testFileFullPath = dir </> file
```

Comment:
This function doesn't appear to be used anywhere.

Reply:
Removed.